src/app/App.js

```diff
@@ -27,6 +29,10 @@ function App() {
 
   useEffect(() => {
     showUser(userId)
+    
+    setTimeout(() => {
+      setIsLoading(false)
+    }, 5000)
```

Comment:
The loading screen is based off of time then? Rather then when the scene is finished loading?